src/sbt-test/sbt-header/simple/build.sbt

```diff
@@ -0,0 +1,21 @@
+import de.heikoseeberger.sbtheader.license.Apache2_0
+
+headers := Map(
+  "scala" -> Apache2_0("2015", "Heiko Seeberger")
+)
+
+TaskKey[Unit]("check") <<= scalaSource in Compile map { (scalaSource) =>
+
+  val actualPath = (scalaSource / "Main.scala").toString
+  val expectedPath = (scalaSource / "Main.scala_expected").toString
```

Comment:
build.sbt only defines a header mapping for scala files. So no, this file won't be affected. Putting the expected files alongside the actual files makes it easier to select them inside the check task.